astro/tests/integration_test.go

```diff
@@ -71,10 +71,18 @@ func stringVersionMatches(v string, versionConstraint string) bool {
 }
 
 // compiles the astro binary and returns the path to it.
-func compileAstro(dir string) (string, error) {
+func compileAstro(dir string, ldflags []string) (string, error) {
```

Comment:
Let's just make this `goBuildFlags []string` and allow people to pass arbitrary flags.